contrib/ci/Dockerfile-fedora.in

```diff
@@ -8,4 +8,4 @@ RUN dnf -y update
 RUN echo fubar > /etc/machine-id
 %%%INSTALL_DEPENDENCIES_COMMAND%%%
 WORKDIR /github/workspace
-CMD ["./contrib/ci/fedora.sh"]
+CMD ["./contrib/ci/fedora.sh"%%%QUBES%%%]
```

Comment:
rather than an argument, can this be done via an environment variable? That's how we pass CI and CI_NETWORK and such in.

Reply:
Deleted. I've added the environment variable `QUBES=true`.